### ISSUE TITLE
Make it clearer that non-github projects are included, and can be included

### ISSUE THREAD

**Issue by mmazanec22** [codeforamerica/brigade-project-index-statusboard#87]
Currently, the language of the statusboard implies that only GitHub projects are included. While that's the case for the majority of brigades, it is not inherently true, as any CSV or JSON file could be an organization's source of truth.

The statusboard should include some description about how the data is gathered so that people can understand where it's coming from, what its limitations are, and what they need to do if they want to see non-coding projects show up in the index. This might beg some questions about how we determine the last updated date of projects listed in a CSV or JSON file.

I wrote up a description for the general documentation push last spring that may be helpful to reference, but has perhaps been overwritten recently? [This is the relevant PR where it was added.](https://github.com/codeforamerica/brigade-project-index/commit/76af7ed5727f24fb1d879feb2cc14286e00b0e69)

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
> I wrote up a description for the general documentation push last spring that may be helpful to reference, but has perhaps been overwritten recently? This is the relevant PR where it was added.

Isn't it moved? https://github.com/codeforamerica/brigade-project-index/blob/master/docs/projects/crawler/contributing/get-indexed.md

**Comment by giosce** [comment]
Yes Jonatan, you are right, that is the same description on how to list projects.
Maybe we can add a link from the Dashboard description (in the same set of docs) to the "Crawler" relevant section.

**Comment by ethanstrominger** [comment]
Proposed solution: Add a link "Add project" which links to the page on github which describes how to add projects.

**Comment by giosce** [comment]
Instructions are already there, we have a 2 hops from ProjectIndex page to detailed instructions.
https://brigade.cloud/projects/crawler/contributing/get-indexed/
